content/en/docs/writing-policies/generate.md

```diff
@@ -77,7 +77,7 @@ spec:
       kind: ConfigMap
       name: zk-kafka-address
       # generate the resource in the new namespace
-      namespace: "{{request.object.metadata.name}}"
+      namespace: "{{request.object.metadata.namespace}}"
```

Comment:
The trigger resource is Namespace, so we need `metadata.name` of the namespace itself.